### PR TITLE
Add saver. Closes #281

### LINKED ISSUE #281: Setup Player Prefs for Master/Music/SoundEffect volumes

### PR DESCRIPTION
**Pull request by WhimsySoftware**

### CODE REVIEW

File: src/ProjectNeon/Assets/Scripts/KeysEnum.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿public enum KeysEnum
+{
+    MusicVolume,
+    SoundVolume,
+    CurrentHero,
```
Comment (SilasReinagel) [inline src/ProjectNeon/Assets/Scripts/KeysEnum.cs]: We will never store game state in the PlayerPrefs. Only options. 

No need for `CurrentHero`.

File: src/ProjectNeon/Assets/Scripts/Saver.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
+﻿using UnityEngine;
+
+
+[CreateAssetMenu(fileName = "New Saver", menuName = "Saver")]
+public class Saver : ScriptableObject
```
Comment (SilasReinagel) [inline src/ProjectNeon/Assets/Scripts/Saver.cs]: Let's rename this to `Settings`. We will only be saving settings from the Options menu.

File: src/ProjectNeon/Assets/Scripts/Saver.cs [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+{
+    // @todo #281: 30min Add metod to check if the one key is used more then ones
+    public static void Save(KeysEnum saveKey, string data)
```
Comment (SilasReinagel) [inline src/ProjectNeon/Assets/Scripts/Saver.cs]: These methods should be private. We only need public methods for gettings and setting SoundVolume and MusicVolume.

Having all of these public methods will lead to error if people don't know which data types are used.

Comment (WhimsySoftware) [reply src/ProjectNeon/Assets/Scripts/Saver.cs]: Wouldn't it be necessary to save a player’s progress if he leaves the game in the middle of the session?

**Pull request merged**